### DISCUSSION TITLE
PokemonAPI

### DISCUSSION THREAD

**Discussion by Anani7RS** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#455] [Q&A]
Hola, disculpen ¿a alguien más les sale esto al tratar de ingresar el nombre de un Pokémon en la api?
![image](https://user-images.githubusercontent.com/99102184/159091128-828a3d20-adb7-4b92-be9b-9f9e62b6582b.png)
Ya intenté cambiando el navegador, en mi lap, en mi celular, chequé que tuviera actualizados mis navegadores, pero parece que me sigue saliendo igual

**Comment by gersondlarosa** [comment]
https://pokeapi.co/
a mi si me da puedes privar esa liga

**Reply by Anani7RS** [reply]
Así como me mandaste el link, le di click y me aparece lo mismo

**Comment by Vitaliquid** [comment]
El sitio https://pokeapi.co/ está funcionando. revisa que tu conexión sea https

**Reply by Anani7RS** [reply]
Continua marcando error

**Comment by doguedogue** [comment]
Si funciona mira

![Screenshot_2022-03-18-17-20-38-543_com.android.chrome.jpg](https://user-images.githubusercontent.com/23409026/159096208-08a5f149-2fdc-43ba-b690-c8c3d9ba2bbd.jpg)
